### DISCUSSION TITLE
Browsing source generator generated files in MSVS 2019 16.9 (or 16.10)

### DISCUSSION THREAD

**Discussion by kiminuo** [dotnet/roslyn#51635] [Q&A] [answered]
Hi guys, 

I have cloned https://github.com/pakrym/jab/ repo and opened `ConsoleSample` project in my MSVS.

There is

```csharp
<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
<CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
```

in https://github.com/pakrym/jab/blob/main/src/samples/ConsoleSample/ConsoleSample.csproj

And now I would love to see those generated files in my Visual Studio. Can I do that somehow?

Basically, I build `ConsoleSample` project and the Jab source generator is invoked and a new file is added to `$(BaseIntermediateOutputPath)\GeneratedFiles` and I would love to add `GeneratedFiles` folder to my MSVS to see how the generated files change over time as I work with the sample project.

Thanks

**Comment by jasonmalinowski** [comment, marked as answer]
We shipped Visual Studio 16.9 yesterday which should show the generated files under the Analyzer node in Visual Studio:

![image](https://user-images.githubusercontent.com/201340/109874835-7f386000-7c24-11eb-9c00-5b023da9afd0.png)

Those will appear whether you have EmitCompilerGeneratedFiles set or not.

**Reply by kiminuo** [reply]
Thanks, I can see it now!
